frontend/src/app/modules/nav/desktop-nav/desktop-nav.component.html

```diff
@@ -1,35 +1,31 @@
-<div class='nav-buttons-container' [withGtmContext]="gtmContext">
-    <ng-container *ngIf="isLoggedIn">
-        <button mat-button routerLink='/interwencje'
-                routerLinkActive="active"
-                [routerLinkActiveOptions]="{exact: true}">
-            <mat-icon>list</mat-icon>
-            Interwencje
-        </button>
-        <button mat-button routerLink='/interwencje/mapa' routerLinkActive="active">
-            <mat-icon>map</mat-icon>
-            Mapa
-        </button>
-        <button mat-button routerLink='/interwencje/zglos' routerLinkActive="active">
-            <mat-icon>notifications_active</mat-icon>
-            Zgłoś
-        </button>
-    </ng-container>
-    <button *ngIf="!isLoggedIn"
-            mat-button routerLink='/zglos-interwencje'
-            routerLinkActive="active">
-        Zgłoś interwencję
+<div class="nav-buttons-container" [withGtmContext]="gtmContext">
+  <ng-container *ngIf="isLoggedIn">
+    <button mat-button routerLink="/interwencje" routerLinkActive="active" [routerLinkActiveOptions]="{ exact: true }">
+      <mat-icon>list</mat-icon>
+      Interwencje
     </button>
-</div>
-<div class='auth-buttons-container' [withGtmContext]="gtmContext">
-    <button *ngIf='!isLoggedIn' class="login" mat-button (click)='logIn.emit()'>
-        <mat-icon>account_circle</mat-icon>
-        Zaloguj
+    <button mat-button routerLink="/interwencje/mapa" routerLinkActive="active">
+      <mat-icon>map</mat-icon>
+      Mapa
+    </button>
+    <button mat-button routerLink="/interwencje/zglos" routerLinkActive="active">
+      <mat-icon>notifications_active</mat-icon>
+      Zgłoś
     </button>
-    <ng-container *ngIf='isLoggedIn'>
-        <span class='greeting'>Witaj, Inspektorze!</span>
-        <button class="logout" mat-button title='Wyloguj' (click)='logOut.emit()'>
-            <mat-icon>exit_to_app</mat-icon>
-        </button>
-    </ng-container>
+  </ng-container>
+  <button *ngIf="!isLoggedIn" mat-button routerLink="/zglos-interwencje" routerLinkActive="active">
+    Zgłoś interwencję
+  </button>
+</div>
+<div class="auth-buttons-container" [withGtmContext]="gtmContext">
```

Comment:
Buttons in this section have been changed to anchors with auth links. This is the only real change in the file - rest of the changes have been done by prettier.